syntax = "proto3";

message MessageWithCorrelationId {
    string event_id = 1;
    string correlation_id = 2;
}

message MessageWithoutCorrelationId {
    string event_id = 1;
}
